client/src/Components/MyApplications/index.jsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Container } from 'react-bootstrap';
+import ApplicationCard from '../CommonComponents/OfferCard';
+import applications from '../utils/myApplications';
+import './style.css';
 
 export default function MyApplications() {
```

Comment:
I think you need to send a request to get my-application list!!? #41 
so you need `componentDidMounth` :thinking:

Reply:
that's right, I proposed to make it when I joined front-end with back-end.
I'll change it now 
thanks